src/transformers/generation_utils.py

```diff
@@ -483,31 +483,6 @@ def _expand_inputs_for_generation(
             model_kwargs["encoder_outputs"] = encoder_outputs
         return input_ids, model_kwargs
 
-    @staticmethod
-    def _init_sequence_length_for_generation(
-        input_ids: torch.LongTensor, max_length: int
-    ) -> Tuple[torch.Tensor, torch.Tensor, int]:
-        unfinished_sequences = input_ids.new(input_ids.shape[0]).fill_(1)
-        sequence_lengths = input_ids.new(input_ids.shape[0]).fill_(max_length)
-
-        cur_len = input_ids.shape[-1]
-        return sequence_lengths, unfinished_sequences, cur_len
```

Comment:
`sequence_lengths` is never actually used anywhere so we can delete it completely